bundle/src/assembly/resources/scenarios/Tiergarten/sumo/sumo_config.json

```diff
@@ -1,3 +1,11 @@
 {
-	"sumoConfigurationFile" : "tiergarten.sumocfg"
+    "sumoConfigurationFile": "tiergarten.sumocfg",
+    "additionalVehicleTypeParameters": {
+        "TLPKW": {
```

Comment:
The prototype names already changed when migrating to MOSAIC, but we forgot to update the rou.xml. The names of the types are now "Car" and "TrafficLightCar".

Reply:
Adapted.